REB_firmware/REB_retrieve_versions_after/v0/ccseoREB_retrieve_versions.py

```diff
@@ -12,14 +12,17 @@
 
 cdir = tsCWD
 
-
```

Comment:
Instead of having a separate harnessed job just to retrieve the installed version, running this jython script could simply be added to the end of the firmware update job.